app/models/processing_destinations_lookup.js

```diff
@@ -0,0 +1,13 @@
+import Model from "ember-data/model";
+import attr from "ember-data/attr";
+
+export default Model.extend({
+  name: attr("string"),
+  lookups: function() {
+    const allRecords = this.store.peekAll("processing_destinations_lookup");
+    return allRecords.map(record => ({
+      id: record.get("id"),
+      name: record.get("name")
+    }));
+  }
```

Comment:
Don't think that `lookups` method has its place in the model (also did a search, doesn't seem to be used?)

Reply:
yeah. I had added it as some experimentation. Then forgot to remove. Thanks, removed it